tests/functional/common.js

```diff
@@ -54,9 +54,9 @@ export function withTempAddonDir(
 }
 
 
-// reportRunCommandError helper
+// reportProgramErrors helper
 
-export function reportRunCommandError(obj: Object, msg: ?string) {
+export function reportProgramErrors(obj: Object, msg: ?string) {
```

Comment:
How about just `reportCommandErrors` and then renaming [runCommand()](https://github.com/shubheksha/web-ext/blob/5f8fc739b7be17a4f076517c04d1dd9cc4ae5824/tests/functional/common.js#L98) to `execCommand()`?